html_text/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+====================
+Text from HTML field
+====================
+
+This module provides some technical features that allow to extract text from
+any chunk of HTML, without HTML tags or attributes. You can chose either:
+
+* To truncate the result by amount of words or characters.
+* To append an ellipsis (or any character(s)) at the end of the result.
+
+It can be used to easily generate excerpts.
+
+Usage
+=====
+
+This module just adds a technical utility, but nothing for the end user.
+
+If you are a developer and need this utility for your module, see these
+examples and read the docs inside the code.
+
+Python example::
+
+    @api.multi
```

Comment:
Maybe remove this from the docs, as it is no longer used.

Reply:
@victoralmau you need to remove it from the `USAGE.rst` file, otherwise it will just be overwritten by the bot

Reply:
In this addon there is no readme folder, you should create it (with the corresponding files) so that the bot correctly regenerates this file when it is merged?.

Reply:
Oh ok. In that case, you should probably follow the 2nd point in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration to create the correct folder